addons/hr_holidays/tests/test_holidays_calendar.py

```diff
@@ -42,3 +42,19 @@ def test_hours_time_off_request_calendar_view(self):
         self.assertEqual(last_leave.date_from.weekday(), 3, "It should be Thursday")
         self.assertEqual(last_leave.date_from.hour, expected_leave_start, "Wrong start of the day")
         self.assertEqual(last_leave.date_to.hour, expected_leave_end, "Wrong end of the day")
+
+    def test_time_type_other(self):
+        leave_type = self.env['hr.leave.type'].create({
```

Comment:
maybe improve the name or add a comment explaining what we are testing, so that when our colleagues add a feature or fix something and this test fails, they can understand what is happening